tee-worker/litentry/core/data-providers/src/lib.rs

```diff
@@ -68,7 +68,7 @@ pub mod twitter_official;
 
 const TIMEOUT: Duration = Duration::from_secs(3u64);
 
-pub const LIT_TOKEN_ADDRESS: &str = "0xb59490aB09A0f526Cc7305822aC65f2Ab12f9723";
+pub const LIT_TOKEN_ADDRESS: &str = "0xb59490ab09a0f526cc7305822ac65f2ab12f9723";
```

Comment:
ethereum addresses should be case insensitive. 
If achainable is case sensitive in this matter, we should remain address the same and instead call `to_lower_case` on it in achainable data-provider as a more complete solution. This will eliminate the risk of this happening again.

Reply:
Yes. This case sensitive topic should be taken care in a bigger round. We should in general check all address case sensitive topic.